systemd/system-generators/coreos-autologin-generator

```diff
@@ -52,6 +64,8 @@ autologin() {
         overlay_unit "getty@.service"
         overlay_unit "serial-getty@.service"
     fi
+
+    emergency_autologin
```

Comment:
To be on the safe side, lets pass "${arg}" as an argument, the function should then exit early if arg is not empty but doesn't exist in /sys/class/tty/console/active

that way we don't open up a tty in emergency mode that wouldn't otherwise be open.